### DISCUSSION TITLE
Something is wrong with this plugin definition and I can't figure out what

### DISCUSSION THREAD

**Discussion by Hubro** [wbthomason/packer.nvim#590] [Q&A] [answered]
I'm trying to use `lualine.vim` with `lsp-status.nvim`:

```lua
  use {
    "hoob3rt/lualine.nvim",
    requires = { "nvim-lua/lsp-status.nvim" },
    config = function()
      print("Setting up lualine!")
      print(vim.inspect(require"lsp-status".status))

      require("lualine").setup {
        tabline = {
          lualine_a = { "filename", require"lsp-status".status }
        }
      }
    end
  }
```

The print statements are there for troubleshooting. They are executed, and the last print statement produces `"<function 1>"`, so the `lsp-status` module appears to be available.

However, when I run this, it produces a blank tabline.

BUT, if I now run `so ~/.config/nvim/plugin/packer_compiled.lua`, the tabline is initialized properly and I can see the LSP status.

I can't for the life of me think of any reason why this setup function doesn't work on startup, but it works if I trigger it again immediately after opening nvim, even in an empty window.

I have no clue if this is a packer issue, a lualine issue or a lsp-status issue. I'm asking this here in case any of you can spot anything wrong with the packer snippet above.

**Comment by Hubro** [comment, marked as answer]
Nope, turns out it's a bug in lualine. Sorry for the noise.

**Reply by wbthomason** [reply]
No worries, and sorry for not replying sooner!
